src/main/java/de/tum/cit/aet/artemis/buildagent/service/BuildAgentInformationService.java

```diff
@@ -52,11 +52,11 @@ public BuildAgentInformationService(BuildAgentConfiguration buildAgentConfigurat
     }
 
     public void updateLocalBuildAgentInformation(boolean isPaused) {
-        updateLocalBuildAgentInformationWithRecentJob(null, isPaused, false);
+        updateLocalBuildAgentInformationWithRecentJob(null, isPaused, false, 0);
```

Comment:
`private static final int DEFAULT_CONSECUTIVE_FAILURES = 0;`

Can we make this a variable instead?